common/src/main/java/io/github/consistencyplus/consistency_plus/registry/CPlusItemGroups.java

```diff
@@ -2,14 +2,16 @@
 
 import dev.architectury.registry.CreativeTabRegistry;
 import io.github.consistencyplus.consistency_plus.base.ConsistencyPlusMain;
+import io.github.consistencyplus.consistency_plus.blocks.BlockTypes;
+import io.github.consistencyplus.consistency_plus.blocks.BlockVariations;
 import net.minecraft.item.Item;
 import net.minecraft.item.ItemGroup;
 import net.minecraft.item.ItemStack;
 
 public class CPlusItemGroups {
-	public static final ItemGroup CONSISTENCY_PLUS_STONES = CreativeTabRegistry.create(ConsistencyPlusMain.id("stone"), () -> new ItemStack(CPlusItems.POLISHED_STONE));
-	public static final ItemGroup CONSISTENCY_PLUS_DYEABLES = CreativeTabRegistry.create(ConsistencyPlusMain.id("dyeable"), () -> new ItemStack(CPlusItems.TERRACOTTA_STAIRS));
-	public static final ItemGroup CONSISTENCY_PLUS_MISC = CreativeTabRegistry.create(ConsistencyPlusMain.id("misc"), () -> new ItemStack(CPlusItems.PURPUR_BLOCK));
+	public static final ItemGroup CONSISTENCY_PLUS_STONES = CreativeTabRegistry.create(ConsistencyPlusMain.id("stone"), () -> new ItemStack(CPlusEnhancedRegistry.getItem("stone", BlockVariations.BLOCK, BlockTypes.POLISHED)));
```

Comment:
can't we use constants from EnhancedRegistry here

Reply:
Do you mean for the strings? I guess so.